### ISSUE TITLE
Marketplace: Breadcrumb with long text padding / wrapping

### ISSUE THREAD

**Issue by cpapazoglou** [Automattic/wp-calypso#62065]
- In your site, go to `Plugins`
- Search for `subscriptions`
- Click on `Paid Memberships Pro`

**Result:**
- The padding between the separators doesn't look great (perhaps adding `text-align: center` can help)
- The breadcrumb wraps and doesn't seem _professional_. Should we truncate it so that it doesn't wrap?

![CleanShot 2022-03-18 at 15 59 08@2x](https://user-images.githubusercontent.com/12430020/159016804-86309667-492f-4865-805e-0afacad31b66.png)

cc: @vinimotaa

**Comment by SaxonF** [comment]
Quick solution here is to either set character limit on breadcrumbs to median length of plugin name or set [text-overflow on name](https://www.w3schools.com/cssref/css3_pr_text-overflow.asp)

**Comment by cpapazoglou** [comment]
Thanks @SaxonF! Just to make it clear, you are suggesting the following design, clipping the plugin name.
![CleanShot 2022-05-09 at 15 53 06@2x](https://user-images.githubusercontent.com/12430020/167413992-903b2a53-6d87-4a55-a5fb-25e570d08bcf.png)

**Comment by SaxonF** [comment]
Yep! Thanks @cpapazoglou . Setting text overflow should allow it to be scalable.

**Comment by gcsecsey** [comment]
Backlog grooming:
Paid plugins: Take up all the space until the billing interval switcher
Free plugins: Take up the full width of the page
